src/layouts/base-new/index.tsx

```diff
@@ -65,15 +64,16 @@ const BaseNewLayout = ({
 }: BaseNewLayoutProps) => {
 	const router = useRouter()
 
-	/** @TODO remove conditional after dark mode is released */
-	const Layout = __config.flags.enable_dark_mode
-		? CoreDevDotLayoutWithTheme
-		: CoreDevDotLayout
+	/**
+	 * @TODO remove conditional after dark mode is released
+	 * Right now we are forcing 'light' mode on all pages
+	 */
+	const Layout = CoreDevDotLayoutWithTheme
 	const layoutProps = __config.flags.enable_dark_mode
 		? {
 				theme,
 		  }
-		: []
+		: { theme: GlobalThemeOption.light }
```

Comment:
@BRKalow another meaningful change I just pushed up here. We need to enable the light theme for the app so these themed images 'work'. Otherwise both images render since the `data-theme` attribute isn't set on the `html` element unless we render the theme provider. There are other ways we could work around it, but this felt the simplest. 

Lmk your thoughts!

Reply:
This seems like a good blanket fix. Initially I had a thought that we should add logic to `ThemedImage`, but then we'd have to do the same for any component that has similar logic. This seems more reasonable, nice work!